csun-metalab/nomi
# NAM-538 -- UI: Croppa Functionality is displayed on the wrong image after default is set

# Description
Fixed modal logic error.
  
# Testing
1. Visit any student profile and set default to avatar (Student Uploaded) image.
2. Close modal, re-open, and upload a photo for likeness (Faculty Uploaded) image.
3. Close modal, re-open, refresh page.
4. Ensure for each test that the croppa-functionality (Upload & Confirm Image) button stay on likeness image only!
5. Repeat steps 1-4 for Gallery Cards.
6. Try to break the functionality.
